docs/process/style_guide.md

```diff
@@ -724,7 +724,7 @@ enum IcuError {
 A couple of crates by `@dtolnay` that are considered "new wave of good error APIs" and are complementary to each other:
```

Comment:
nit: by dtolnay and yaahc

Reply:
Merging now and I will take care of this in #1254 so that I don't have to wait for CI again